docs/modules/sql/pages/sql-reflection-configuration.adoc

```diff
@@ -0,0 +1,96 @@
+= Configuring Reflection
+:description: Clusters can be configured to restrict the capability of creating objects through reflection during SQL execution.
```

Comment:
```suggestion
:description: Clusters can be configured to restrict the capability of creating objects through reflection during SQL execution (applicable to INSERT/SINK/UPDATE operations).
```

Reply:
this is tricky. Currently you cannot create UDT instance in SELECT projection (currently projections produce only `ROW` type), but in theory is it possible  and one day might be supported.
I think we can leave the description as is.